internal/services/eventgrid/eventgrid_domain_data_source_test.go

```diff
@@ -0,0 +1,126 @@
+package eventgrid_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance/check"
+)
+
+type EventGridDomainDataSource struct {
+}
+
+func TestAccEventGridDomainDataSource_basic(t *testing.T) {
```

Comment:
we can probably remove the other tests, usually we just have a single data source test